README.md

```diff
@@ -543,6 +575,9 @@ request. After setting errors for an pb_utils.InferenceResponse
 object, use InferenceResponseSender.send() to send response with the
 error back to the user.
 
+Starting from 23.10, request cancellation can be checked directly on the
+`InferenceResponseSender` object using `response_sender.is_cancelled()`.
```

Comment:
We should mention that the user need to explicitly send a final flag even if the request is cancelled.

Reply:
Sure. [Decoupled final flag docs update](https://github.com/triton-inference-server/python_backend/pull/304/commits/09e358adab39322dbf807d00a9c993f16c375edf)